README.md

```
@@ -133,7 +131,15 @@ These commands are recommended to be run manually once after initial setup (Ansi
 ### LLM Tools Configuration
 
 - Run `just llm` to configure Node.js-based LLM tools (Claude, Gemini, Codex).
-- This ensures necessary configuration directories and files are created and symlinked from `ansible/roles/nodejs/config/common`.
+- This ensures necessary configuration directories and files are created and symlinked from `ansible/roles/nodejs/config/common` to your home directory.
+
+**Project Context Initialization**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)